### PR TITLE
`azurerm_app_service_slot`  - add test for action/name/priority/service_tag/headers.x_azure_fdid/x_fd_health_probe/x_forwarded_for/x_forwarded_host

### PR DESCRIPTION
**Pull request by shu-ying789**
s2

### CODE REVIEW

**Review by tombuildsstuff — APPROVED** [review]
LGTM 👍 - although note that this resource will be superseded in 3.0: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/guides/3.0-app-service-beta

File: internal/services/web/app_service_slot_resource_test.go [hunk 1]
```diff
@@ -1677,6 +1692,59 @@ resource "azurerm_app_service_slot" "test" {
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger, data.RandomInteger, data.RandomInteger)
 }
 
+func (r AppServiceSlotResource) siteConfig(data acceptance.TestData) string {
```
Comment (tombuildsstuff) [inline internal/services/web/app_service_slot_resource_test.go:1695]:
```suggestion
func (r AppServiceSlotResource) ipRestrictionHeaders(data acceptance.TestData) string {
```

File: internal/services/web/app_service_slot_resource_test.go [hunk 2]
```diff
@@ -194,6 +194,21 @@ func TestAccAppServiceSlot_storageAccounts(t *testing.T) {
 	})
 }
 
+func TestAccAppServiceSlot_siteConfig(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_app_service_slot", "test")
+	r := AppServiceSlotResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.siteConfig(data),
```
Comment (tombuildsstuff) [inline internal/services/web/app_service_slot_resource_test.go:203]:
```suggestion
			Config: r.ipRestrictionHeaders(data),
```

File: internal/services/web/app_service_slot_resource_test.go [hunk 3]
```diff
@@ -194,6 +194,21 @@ func TestAccAppServiceSlot_storageAccounts(t *testing.T) {
 	})
 }
 
+func TestAccAppServiceSlot_siteConfig(t *testing.T) {
```
Comment (tombuildsstuff) [inline internal/services/web/app_service_slot_resource_test.go:197]:
```suggestion
func TestAccAppServiceSlot_ipRestrictionHeaders(t *testing.T) {
```

**Review by tombuildsstuff — APPROVED** [review]
@jackofallops pointed out this test should be renamed to match the fields too

**Review by katbyte — APPROVED** [review]
LGTM 🦀

**Pull request merged**